docs/data-discovery/tracking-catalog/index.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Data Discovery through the Tracking Catalog"
+date: "2022-10-26"
+sidebar_position: 0
+---
+
+
+## Introducing the Tracking Catalog
+
+
+Discovery and understanding of the data your organisation is tracking through Snowplow is essential to enable teams to unlock real value from that data.
+We know that in many cases, Data Analysts, Data Scientists and Product Managers struggle to find out what is being tracked, and what the data means. They normally have to rely on tribal knowledge and source this information directly with the front end developers who instrumented the tracking, or raise a request to the Data Team.
+
+However, through the Tracking Catalog, all users can have a holistic view of all their tracking in a always up-to-date and easy to consume view.
+
+
+* * *
+
+
+## Browsing the Tracking Catalog
+
+
+**_NOTE:_** Please note that the Tracking Catalog is available only for BDP Customers on selected Tiers. If you're unsure about your eligibility, please get in touch with your Customer Success Manager.
+
+Once logged in Snowplow Console, you will see a new link on the left hand side navigation, called Tracking Catalog. 
+
+
+![](images/Tracking-Catalog-Nav.png)
+
+
+Once you navigate to it, you will be presented with a view showing all the Events and Entities that have been tracked your Snowplow production pipelines in the past 90 days.
+
+![](images/TC-Landing-page.png)
+
+
+If you select a particular Event, you can see the details of the Events, what properties are tracked for this event, described in plain English therefore easily consumable by a non-technical audience. You can also see any validation criteria associated with this event. The catalog will also display all canonical properties which are also tracked out of the box, when you use Snowplow trackers.
+
+![](images/event-details.png)
+
+
+![](images/canonical-properties.png)
```

Comment:
Just noticed one other little thing, the back to back images are a little tricky to look at when looking at the doc page, I'd perhaps suggest moving the sentence from the above paragraph to be in between these images:

> The catalog will also display all canonical properties which are also tracked out of the box, when you use Snowplow trackers.

Reply:
done